### PR TITLE
[strings] More clear hints when building a route

### PR DESCRIPTION
**Pull request by biodranik**
Now some new users are confused and do not understand that they should tap/select something on the map to build a route. This change should provide a better hint.

@Jean-BaptisteC can you please check if the longest translation fits/looks well with the smallest device width?

### CODE REVIEW

File: data/strings/strings.txt [hunk 1]
```diff
@@ -20153,93 +20153,95 @@
...
-    sk = Pridaním počiatočného bodu začnite plánovať trasu
-    sv = Lägg till startpunkt för att planera en rutt
-    sw = Ongeza eneo la kuanzia ili kupanga njia ya kufuata
-    th = เพิ่มจุดเริ่มต้นเพื่อวางแผนเส้นทาง
-    tr = Rotayı planlamak için bir başlangıç noktası ekleyin
-    uk = Вкажіть початкову точку для побудови маршруту
-    vi = Bổ sung điểm bắt đầu để lập kế hoạch lộ trình
-    zh-Hans = 添加起点以规划路线
-    zh-Hant = 新增起點以計劃路線
+    comment = User selected the destination by pressing Route To, but the current position is unknown. User needs to select a starting point of a route using search or by tapping on the map and then pressing "Route From".
```
Comment (biodranik) [inline data/strings/strings.txt:20156]: @organicmaps/translations can you please help with proof-reading this translation in your language (or provide a clearer version)?

File: data/strings/strings.txt [hunk 2]
```diff
@@ -20153,93 +20153,95 @@
...
-    sk = Pridaním cieľového bodu naplánujete trasu
-    sv = Lägg till slutpunkt för att planera en rutt
-    sw = Ongeza eneo la kumalizia ili kupanga njia ya kufuata
-    th = เพิ่มจุดสิ้นสุดเพื่อวางแผนเส้นทาง
-    tr = Rotayı planlamak için bir varış noktası ekleyin
-    uk = Вкажіть кінцеву точку для побудови маршруту
-    vi = Bổ sung điểm kết thúc để lập kế hoạch lộ trình
-    zh-Hans = 添加终点以规划路线
-    zh-Hant = 新增終點以計劃路線
+    comment = User selected the start of a route by pressing Route From. Now the destination of a route should be selected using search or by tapping on the map and then pressing "Route To".
```
Comment (biodranik) [inline data/strings/strings.txt:20202]: @organicmaps/translations can you please help with proof-reading this translation in your language (or provide a clearer version)?

File: data/strings/strings.txt [hunk 3]
```diff
@@ -20153,93 +20153,95 @@
...
+    ja = 検索を使うか、地図上をタップしてルートの出発点を追加する
+    ko = 검색을 사용하거나 지도를 탭하여 경로 시작점을 추가합니다
+    lt = Naudokite paiešką arba bakstelėkite žemėlapyje, kad pridėtumėte maršruto pradžios tašką
+    mr = मार्ग प्रारंभ बिंदू जोडण्यासाठी नकाशावर शोध वापरा किंवा टॅप करा
+    nb = Bruk søk eller trykk på kartet for å legge til et startpunkt for ruten
+    nl = Gebruik zoeken of tik op de kaart om een startpunt voor de route toe te voegen
+    pl = Użyj funkcji wyszukiwania lub dotknij mapy, aby dodać punkt początkowy trasy
+    pt = Utiliza a pesquisa ou toca no mapa para adicionar um ponto de partida do percurso
+    pt-BR = Use a busca ou toque no mapa para adicionar um ponto de partida da rota
+    ro = Utilizați funcția de căutare sau atingeți ușor pe hartă pentru a adăuga un punct de plecare al traseului
```
Comment (vgrigoriu) [inline data/strings/strings.txt:20188]: You can leave off the "al traseului" at the end, to make it symmetrical with the one below.

File: data/strings/strings.txt [hunk 4]
```diff
@@ -20153,93 +20153,95 @@
...
+    da = Brug søgning eller tryk på kortet for at tilføje et startpunkt for en rute
+    de = Verwende die Suche oder tippe auf die Karte, um einen Routenstartpunkt hinzuzufügen
+    el = Χρησιμοποιήστε την αναζήτηση ή πατήστε στο χάρτη για να προσθέσετε ένα σημείο εκκίνησης της διαδρομής
+    es = Utiliza la búsqueda o toca en el mapa para añadir un punto de inicio de ruta
+    et = Marsruudi alguspunkti lisamiseks kasutage otsingut või puudutage kaarti
+    eu = Erabili bilaketa edo sakatu mapan ibilbidearen abiapuntua gehitzeko
+    fa = برای افزودن نقطه شروع مسیر، از جستجو استفاده کنید یا روی نقشه ضربه بزنید
+    fi = Käytä hakua tai napauta karttaa lisätäksesi reitin lähtöpisteen
+    fr = Utilise la recherche ou tape sur la carte pour ajouter un point de départ d'itinéraire
+    he = השתמש בחיפוש או הקש על המפה כדי להוסיף נקודת התחלה למסלול
```
Comment (NeatNit) [inline data/strings/strings.txt:20174]:
```suggestion
    he = יש להוסיף נקודת התחלה למסלול באמצעות חיפוש או בלחיצה על המפה
```

File: data/strings/strings.txt [hunk 5]
```diff
@@ -20153,93 +20153,95 @@
...
+    da = Brug søgning eller tryk på kortet for at tilføje et destinationspunkt
+    de = Verwende die Suche oder tippe auf die Karte, um einen Zielpunkt hinzuzufügen
+    el = Χρησιμοποιήστε την αναζήτηση ή πατήστε στο χάρτη για να προσθέσετε ένα σημείο προορισμού
+    es = Utiliza la búsqueda o toca en el mapa para añadir un punto de destino
+    et = Kasutage otsingut või koputage kaardil sihtkoha lisamiseks
+    eu = Erabili bilaketa edo sakatu mapan helmuga puntu bat gehitzeko
+    fa = برای افزودن نقطه مقصد، از جستجو استفاده کنید یا روی نقشه ضربه بزنید
+    fi = Käytä hakua tai napauta karttaa lisätäksesi kohdepisteen.
+    fr = Utilise la recherche ou tape sur la carte pour ajouter un point de destination
+    he = השתמש בחיפוש או הקש על המפה כדי להוסיף נקודת יעד
```
Comment (NeatNit) [inline data/strings/strings.txt:20220]:
```suggestion
    he = יש לבחור נקודת יעד באמצעות חיפוש או בלחיצה על המפה
```

**Review by jimcarst — APPROVED** [review]
nl ok

File: data/strings/strings.txt [hunk 6]
```diff
@@ -20153,93 +20153,95 @@
...
+    it = Usa la ricerca o tocca la mappa per aggiungere un punto di destinazione
+    ja = 検索を使うか、地図上をタップして目的地を追加する
+    ko = 검색을 사용하거나 지도를 탭하여 목적지 지점을 추가합니다
+    lt = Naudokite paiešką arba bakstelėkite žemėlapyje, kad pridėtumėte paskirties tašką
+    mr = गंतव्य बिंदू जोडण्यासाठी नकाशावर शोधा किंवा टॅप करा
+    nb = Bruk søk eller trykk på kartet for å legge til et destinasjonspunkt
+    nl = Gebruik Zoeken of tik op de kaart om een bestemmingspunt toe te voegen
+    pl = Użyj funkcji wyszukiwania lub dotknij mapy, aby dodać punkt docelowy
+    pt = Utiliza a pesquisa ou toca no mapa para adicionar um ponto de destino
+    pt-BR = Use a busca ou toque no mapa para adicionar um ponto de destino
```
Comment (matheusgomesms) [inline data/strings/strings.txt:20233]:
```suggestion
    pt = Use a busca ou toque no mapa para adicionar um ponto de destino
```

File: data/strings/strings.txt [hunk 7]
```diff
@@ -20153,93 +20153,95 @@
...
+    it = Usa la ricerca o tocca la mappa per aggiungere un punto di partenza del percorso
+    ja = 検索を使うか、地図上をタップしてルートの出発点を追加する
+    ko = 검색을 사용하거나 지도를 탭하여 경로 시작점을 추가합니다
+    lt = Naudokite paiešką arba bakstelėkite žemėlapyje, kad pridėtumėte maršruto pradžios tašką
+    mr = मार्ग प्रारंभ बिंदू जोडण्यासाठी नकाशावर शोध वापरा किंवा टॅप करा
+    nb = Bruk søk eller trykk på kartet for å legge til et startpunkt for ruten
+    nl = Gebruik zoeken of tik op de kaart om een startpunt voor de route toe te voegen
+    pl = Użyj funkcji wyszukiwania lub dotknij mapy, aby dodać punkt początkowy trasy
+    pt = Utiliza a pesquisa ou toca no mapa para adicionar um ponto de partida do percurso
+    pt-BR = Use a busca ou toque no mapa para adicionar um ponto de partida da rota
```
Comment (matheusgomesms) [inline data/strings/strings.txt:20187]:
```suggestion
    pt = Use a busca ou toque no mapa para adicionar um ponto de partida do percurso
    pt-BR = Use a busca ou toque no mapa para adicionar um ponto de partida da rota
```

**Review by matheusgomesms — CHANGES_REQUESTED** [review]
PT and PT-BR reviewed

File: android/app/src/main/res/values-es/strings.xml [hunk 8]
```diff
@@ -587,8 +587,10 @@
 	<string name="learn_more">Leer más</string>
 	<!-- Subway exits for public transport marks on the map -->
 	<string name="core_exit">Salida</string>
-	<string name="routing_add_start_point">Buscar un origen para el plan de ruta</string>
-	<string name="routing_add_finish_point">Buscar un destino para el plan de ruta</string>
+	<!-- User selected the destination by pressing Route To, but the current position is unknown. User needs to select a starting point of a route using search or by tapping on the map and then pressing "Route From". -->
+	<string name="routing_add_start_point">Utiliza la búsqueda o toca en el mapa para añadir un punto de inicio de ruta</string>
```
Comment (muralito) [inline android/app/src/main/res/values-es/strings.xml:591]: I would keep it simple: "Utiliza la búsqueda o toca en el mapa para añadir el origen de la ruta"

Comment (muralito) [reply android/app/src/main/res/values-es/strings.xml:591]: Or more simple: "Busca o toca en el mapa para añadir el origen de la ruta"

File: android/app/src/main/res/values-es/strings.xml [hunk 9]
```diff
@@ -587,8 +587,10 @@
...
+	<!-- User selected the start of a route by pressing Route From. Now the destination of a route should be selected using search or by tapping on the map and then pressing "Route To". -->
+	<string name="routing_add_finish_point">Utiliza la búsqueda o toca en el mapa para añadir un punto de destino</string>
```
Comment (muralito) [inline android/app/src/main/res/values-es/strings.xml:593]: I would keep it simple: "Utiliza la búsqueda o toca en el mapa para añadir el destino",
 or "Busca o toca en el mapa para añadir el destino"

File: data/strings/strings.txt [hunk 10]
```diff
@@ -20153,93 +20153,95 @@
...
+    ar = استخدام البحث أو النقر على الخريطة لإضافة نقطة وجهة
+    az = Təyinat nöqtəsi əlavə etmək üçün axtarışdan istifadə edin və ya xəritədə klikləyin
+    be = Скарыстайцеся пошукам або націсніце на мапе, каб дадаць пункт прызначэння
+    bg = Използвайте търсене или докоснете картата, за да добавите точка на дестинация
+    ca = Utilitzeu la cerca o toqueu el mapa per afegir un punt de destinació
+    cs = Pomocí vyhledávání nebo klepnutím na mapu přidejte cílový bod
+    da = Brug søgning eller tryk på kortet for at tilføje et destinationspunkt
+    de = Verwende die Suche oder tippe auf die Karte, um einen Zielpunkt hinzuzufügen
+    el = Χρησιμοποιήστε την αναζήτηση ή πατήστε στο χάρτη για να προσθέσετε ένα σημείο προορισμού
+    es = Utiliza la búsqueda o toca en el mapa para añadir un punto de destino
```
Comment (muralito) [inline data/strings/strings.txt:20214]: es=Busca o toca en el mapa para añadir el destino

File: data/strings/strings.txt [hunk 11]
```diff
@@ -20153,93 +20153,95 @@
...
+    ar = استخدام البحث أو النقر على الخريطة لإضافة نقطة بداية الطريق
+    az = Marşrutun başlanğıc nöqtəsi əlavə etmək üçün axtarışdan istifadə edin və ya xəritədə klikləyin
+    be = Скарыстайцеся пошукам або націсніце на мапе, каб дадаць пачатак маршруту
+    bg = Използвайте търсене или докоснете картата, за да добавите начална точка на маршрута
+    ca = Utilitzeu la cerca o toqueu el mapa per afegir un punt de partida de la ruta
+    cs = Pomocí vyhledávání nebo klepnutím na mapu přidejte výchozí bod trasy
+    da = Brug søgning eller tryk på kortet for at tilføje et startpunkt for en rute
+    de = Verwende die Suche oder tippe auf die Karte, um einen Routenstartpunkt hinzuzufügen
+    el = Χρησιμοποιήστε την αναζήτηση ή πατήστε στο χάρτη για να προσθέσετε ένα σημείο εκκίνησης της διαδρομής
+    es = Utiliza la búsqueda o toca en el mapa para añadir un punto de inicio de ruta
```
[the remark this answers is not in the document]
Comment (biodranik) [reply data/strings/strings.txt:20168]: @fitojb  So what is the final phrase?

File: data/strings/strings.txt [hunk 12]
```diff
@@ -20153,93 +20153,95 @@
...
+    cs = Pomocí vyhledávání nebo klepnutím na mapu přidejte výchozí bod trasy
+    da = Brug søgning eller tryk på kortet for at tilføje et startpunkt for en rute
+    de = Verwende die Suche oder tippe auf die Karte, um einen Routenstartpunkt hinzuzufügen
+    el = Χρησιμοποιήστε την αναζήτηση ή πατήστε στο χάρτη για να προσθέσετε ένα σημείο εκκίνησης της διαδρομής
+    es = Busca o toca en el mapa para añadir el origen de la ruta
+    et = Marsruudi alguspunkti lisamiseks kasutage otsingut või puudutage kaarti
+    eu = Erabili bilaketa edo sakatu mapan ibilbidearen abiapuntua gehitzeko
+    fa = برای افزودن نقطه شروع مسیر، از جستجو استفاده کنید یا روی نقشه ضربه بزنید
+    fi = Käytä hakua tai napauta karttaa lisätäksesi reitin lähtöpisteen
+    fr = Utilise la recherche ou tape sur la carte pour ajouter un point de départ d'itinéraire
```
Comment (Jean-BaptisteC) [inline data/strings/strings.txt:20173]: fr = Utilise la recherche ou appuie sur la carte pour ajouter un point de départ

**Review by Jean-BaptisteC — APPROVED** [review]
✅ Pixel 6 - Android 14
Text fit properly with max text size

**Review by vng — APPROVED** [review]

**Pull request merged**